skrifa/src/outline/glyf/hint/graphics_state/zone.rs

```diff
@@ -401,13 +428,13 @@ impl GraphicsState<'_> {
         point_ix: usize,
         distance: i32,
     ) -> Result<(), HintErrorKind> {
+        // Note: we never adjust x in backward compatibility mode and we never
+        // adjust y in backward compability mode after IUP has been done in
+        // both directions.
```

Comment:
It would be nice if the note said why. The inconsistent behavior is a bit weird.

Reply:
Added a small comment describing motivation and a link to https://learn.microsoft.com/en-us/typography/cleartype/truetypecleartype that has the (very) long-winded explanation.